src/TraceEvent/TraceLoggingEventId.cs

```diff
@@ -10,19 +11,20 @@ namespace Microsoft.Diagnostics.Tracing
     /// <summary>
     /// TraceLoggingEvnetId is a class that manages assigning event IDs (small 64k numbers)
     /// to TraceLogging Style events (which don't have them).  Because TraceEvent uses EventIDs
-    /// so fundamentally this difficiency is very problematic.   
+    /// so fundamentally this deficiency is very problematic.   
```

Comment:
I agree. As you said, the scope of this PR is very small, so I combined the changes. I will follow the guidance in the future. 
Can you also add the guidance to https://github.com/Microsoft/perfview/blob/master/CONTRIBUTING.md